spring-cloud-sleuth-samples/spring-cloud-sleuth-sample-messaging/pom.xml

```diff
@@ -81,6 +81,10 @@
 			<artifactId>lombok</artifactId>
 		</dependency>
 		<dependency>
+			<groupId>com.github.kristofa</groupId>
```

Comment:
This sample should not require brave. I though the intention of this change was to remove the brave dependency everywhere, but this is the least likely place to see one.